### PR TITLE
Add helper text when search with no dictionaries enabled

### PR DESCRIPTION
**Pull request by khaitruong922**
#1029

- When user searches without a dictionary enabled on Search page, it will show a helper message and a hyperlink to the Dictionaries section in the Settings page.

### CODE REVIEW

File: ext/popup.html [hunk 1]
```diff
@@ -45,6 +45,12 @@
                         </div>
                     </div>
 
+                    <div id="no-dictionaries" hidden>
+                        <div class="entry">
+                            <p>No dictionaries have been installed or enabled yet.</p>
+                            <p><a href="#">Go to Dictionaries settings.</a></p>
```
Comment (jamesmaa) [inline ext/popup.html:51]: This url doesn't seem correct?

Comment (khaitruong922) [reply ext/popup.html:51]: Tapping the href which links to dictionary section in the Settings page will break the popup. I'm trying to find how to override this behavior.

I'm not sure if this popup ever shows up if user does not have any dictionary.

Comment (jamesmaa) [reply ext/popup.html:51]: > I'm not sure if this popup ever shows up if user does not have any dictionary.
image

it's possible by doing a lookup via the context menu but it's kind of rare

Comment (khaitruong922) [reply ext/popup.html]: Just checked it again. It only breaks when I don't put target="__blank".

**Review by jamesmaa — APPROVED** [review]

**Pull request merged**